leetcode/src/274.c

```diff
@@ -0,0 +1,20 @@
+int comparer(const int* i, const int* j)

```

Comment:
```suggestion
int diff(const int* i, const int* j)

```

might make more sense here no?